File: aat-android/src/main/java/ch/bailu/aat/gpx/xml_parser/parser/gpx/RteptParser.java
package ch.bailu.aat.gpx.xml_parser.parser.gpx;

import org.xmlpull.v1.XmlPullParser;

import java.io.IOException;

import ch.bailu.aat.gpx.GpxConstants;
import ch.bailu.aat.gpx.xml_parser.scanner.Scanner;

public class RteptParser extends PntParser{
    public RteptParser() {
        super(GpxConstants.QNAME_ROUTE_POINT);
    }

    @Override
    protected void parsed(XmlPullParser parser, Scanner scanner) throws IOException {
        scanner.routeParsed.onHavePoint();
    }
}

File: aat-lib/src/main/java/ch/bailu/aat_lib/xml/parser/gpx/RteptParser.kt
package ch.bailu.aat_lib.xml.parser.gpx

import ch.bailu.aat_lib.gpx.GpxConstants
import ch.bailu.aat_lib.xml.parser.scanner.Scanner
import org.xmlpull.v1.XmlPullParser
import java.io.IOException

class RteptParser : PntParser(GpxConstants.QNAME_ROUTE_POINT) {

    @Throws(IOException::class)
    override fun parsed(parser: XmlPullParser, scanner: Scanner) {
        scanner.routeParsed.onHavePoint()
    }
}
